driver/controller.go

```diff
@@ -425,18 +424,17 @@ func (d *Driver) ControllerUnpublishVolume(ctx context.Context, req *csi.Control
 					log.WithFields(logrus.Fields{
 						"error": err,
 						"resp":  resp,
-					}).Warn("assuming volume is detached already")
+					}).Warn("assuming volume is detached because of error response")
```

Comment:
🤷‍♂ - This might be more clear if it said something like "assuming volume is detached; api reports 'Attachment not found'".  Good with it as is though.

Reply:
My thinking here was that the `error` field we include in the log output includes the reason which our log message points at. The intention was to avoid repetition.

Reply:
👍